bsp/stm32f407_renode/startup.c

```
@@ -0,0 +1,68 @@
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Minimal startup code for STM32F407 on Renode.
+ *
+ * Provides the vector table, Reset_Handler (zeroes BSS, copies data section,
+ * calls main), and default fault handlers.  No clock or peripheral init is
+ * needed because Renode does not require real hardware initialization.
+ ********************************************************************************/
+
+#include <stdint.h>
+#include <string.h>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)